packages/firestore/rollup.config.bundle.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import pkg from './package.json';
+
+const util = require('./rollup.shared');
+
+export default [
+  {
+    input: 'index.bundle.ts',
+    output: {
+      file: pkg.bundle,
+      format: 'es',
+      sourcemap: true
+    },
+    plugins: util.es2017Plugins('browser', /* mangled= */ true),
+    external: util.resolveBrowserExterns,
```

Comment:
This line is the reason for the size of the export. We somehow need to treat all the Firestore-internal code as external, while keeping  the bundle-only exports  internal. I am honestly not sure how to proceed here - maybe @Feiyang1 has a guide or a link that helps.